arduino/resources/checksums.go

```diff
@@ -163,5 +175,9 @@ func CheckDirChecksum(root string) (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	return file.Checksum == checksum, nil
+	if file.Checksum != checksum {
+		return false, fmt.Errorf("Checksum differs from checksum in package.json")
```

Comment:
Please wait before merge. I think `package.json` is not correct here.

```suggestion
		return false, fmt.Errorf("Checksum differs from checksum in package index")
```

Reply:
Mmmh, I think it's correct as it is. You compare the checksum read from `package.json` and the one you calculate given the `root` so the error makes sense.

Reply:
Ok :)